_data/_notifications-template.yml

```diff
@@ -0,0 +1,6 @@
+# Notification Banner example
+Instrumenting Sentry:
+    background: '#141845'
+    textcolor: '#fff'
+    title: Instrumenting Sentry
+    description: Instrumenting 1 Sentry for your backend project? Join us July 11th, at 10 AM PT for the Backend Error Monitoring 101 livestream. <a href="https://go.memfault.com/how-to-debug-android-devices-production">Register now.</a>
```

Comment:
This template will be nice for the future when we may need it -- thank you for adding it! Only note - could you add a new line at the end of this file?